frontend/src/components/dashboard/components/categories-list/categories-list.tsx

```diff
@@ -8,11 +10,31 @@ type Props = {
 };
 
 const CategoriesList: FC<Props> = ({ items }) => {
+  const { searchParams, performSearch } = useSearch();
+
+  const activeCategory = searchParams.get(SearchValues.CATEGORY) ?? '';
+
+  const handleClick = (evt: React.MouseEvent): void => {
```

Comment:
What about the following?
```suggestion
  const handleClick = (keyName: string): void => {
```
And in `Category` component you can:
```
const handleClick = () => onClick(keyName)
```
I don't actually like the solution with event object
Coaches, what do you think about that?